logging/logging.go

```diff
@@ -87,9 +87,7 @@ func (l *StandardLogger) getFields() map[string]interface{} {
 func (l *StandardLogger) SetLevel(level Level) {
 	var logrusLevel logrus.Level
 	switch level {
-	case Error:
-		logrusLevel = logrus.ErrorLevel
-	case Warn:
```

Comment:
While it's not possible to set the log level to `Warn` via CLI options, I think one may still do this via the Go API.. so we might want to keep this case as well.